### PR TITLE
CIV-2141 - Tech Debt - Dynamic Ttles

### PR DESCRIPTION
**Pull request by commitBlob**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-2141

### Change description ###
Add page titles and translations to all missing pages

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "Claim number": "Mialc rebmun",
   "Claimant name": "Tnamialc eman",
   "Claim amount": "Mialc tnuoma",
+  "CLAIM_AMOUNT": "Claim amount",
```
Comment (laarmada) [inline src/main/modules/i18n/locales/cy.json:6]: "Claim amount": "Mialc tnuoma",
  "CLAIM_AMOUNT": "Claim amount",

???? is duplicate? we should fix that in the translation issues?

**Review by laarmada — APPROVED** [review]

File: src/test/unit/views/features/mediation/free-telephone-mediation.test.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import {
 const jsdom = require('jsdom');
 const {JSDOM} = jsdom;
 const govukBodyClass = 'govuk-body';
-const pageTitle = 'Free telephone mediation';
+const pageTitle = 'Free Telephone Mediation';
```
Comment (taomoh) [inline src/test/unit/views/features/mediation/free-telephone-mediation.test.ts:14]: I thought the standard case used in CMC is Sentence case and not Title Case

File: src/test/unit/views/features/response/admission/partialAdmission/already-paid.test.ts [hunk 3]
```diff
@@ -28,6 +28,9 @@ describe('Already Paid View', () => {
       htmlDocument = dom.window.document;
     });
 
+    it('should have correct page title', () => {
+      expect(htmlDocument.title).toEqual('Your money claims account - Already Paid');
```
Comment (taomoh) [inline src/test/unit/views/features/response/admission/partialAdmission/already-paid.test.ts:32]: Should this also be Sentence case?

File: src/test/unit/views/features/response/eligibility/send-your-response-by-email.test.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ import { TestMessages } from '../../../../../utils/errorMessageTestConstants';
 const jsdom = require('jsdom');
 const {JSDOM} = jsdom;
 const govukBodyClass = 'govuk-body';
-const pageTitle = 'Money Claims';
+const pageTitle = 'Send Your Response By Email';
```
Comment (taomoh) [inline src/test/unit/views/features/response/eligibility/send-your-response-by-email.test.ts:12]: Should this also be Sentence case?

**Review by taomoh — COMMENTED** [review]
It looks like the page-title.njk macro still uses English wording as translation key. Should this be modified as well?
Overall good work!
I like how your ordered the entries in the translation files alphabetically. I just hope we're going to stick to that 😃 
I like how using the page title artefact made testing simple by just asserting htmlDocument.title exact value.

File: src/test/unit/views/features/mediation/free-telephone-mediation.test.ts (hunk 2 above)
Comment (commitBlob) [reply src/test/unit/views/features/mediation/free-telephone-mediation.test.ts:14]: our page headers are sentence case
However titles of pages and articles are usually uppercases for each word can convert to sentence case

**Review by taomoh — APPROVED** [review]

**Pull request merged**